openfl/component/aggregator/aggregator.py

```diff
@@ -545,13 +545,28 @@ def _straggler_cutoff_time_elapsed(self) -> None:
             # Check if minimum collaborators reported results
             self._end_of_round_with_stragglers_check()
 
+    def _check_tags(self, tags: tuple[str, ...], allowed_col: str) -> bool:
+        """
+        Check if all tags are either the allowed column or unauthorized.
```

Comment:
"col" should refer to collaborator, rather than column, right?

Reply:
Fixed